misk/src/main/kotlin/misk/web/dashboard/AdminDashboardModule.kt

```diff
@@ -160,7 +166,12 @@ class AdminDashboardTestingModule : KAbstractModule() {
         )
       )
     )
-    install(AdminDashboardModule(true))
+    install(
+      AdminDashboardModule(
+        isDevelopment = true,
+        dashboardProtobufDocUrlPrefix = "https://example.com/"
```

Comment:
Testing module is used in local development so we probably still want Skim to provide the right thing here